resources/asciidoctor/lib/copy_images/extension.rb

```diff
@@ -0,0 +1,69 @@
+require 'csv'
```

Comment:
We have to parse a comma separated list and the simplest way looks to be to use the built in CSV parsing....